### PR TITLE
Refactor TestReconcile_RemotePipelineRef bundle resolver test case

### PR DESCRIPTION
**Pull request by JeromeJu**
# Changes

This commit refactors the v1beta1 bundle syntax which is deprecated in v1 for the TestReconcile_RemotePipelineRef test
case. It moves the taskref and pipelineref off OCI bundle since the test case is based on a remote pipelineRef.

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [n/a] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) if any changes are user facing, including updates to minimum requirements e.g. Kubernetes version bumps
- [x] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [n/a] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings). See some examples of [good release notes](https://github.com/tektoncd/community/blob/main/standards.md#good-release-notes).
- [n/a] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by Yongxuanzhang — APPROVED** [review]

**Comment by Yongxuanzhang** [comment]
I think this PR goes to v1 swap's work? Maybe link it to the issue where we track the progress. 
And let me know when we need a lgtm. :D

File: pkg/reconciler/pipelinerun/pipelinerun_test.go [hunk 1]
```diff
@@ -6463,44 +6454,26 @@ func TestReconcile_RemotePipelineRef(t *testing.T) {
...
   - name: unit-test-1
     taskRef:
-      bundle: %s
-      name: unit-test-task
-`, ref))
-	cms := []*corev1.ConfigMap{withOCIBundles(newFeatureFlagsConfigMap())}
+      resolver: bar
+`)
 
 	// This task will be uploaded along with the pipeline definition.
```
Comment (lbernick) [inline pkg/reconciler/pipelinerun/pipelinerun_test.go:6478]: this comment can be removed

**Review by lbernick — APPROVED** [review]

**Comment by JeromeJu** [comment]
This would be a cleanup for #6444 
thanks for the suggestion @Yongxuanzhang

**Pull request merged**